### PR TITLE
Issue #92: Add Non-Treewalker filter tests to cover more cases(FileTabCharacter)

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #92: Add Non-Treewalker filter tests to cover more cases(FileTabCharacter)

finish left 1, 2 in https://github.com/checkstyle/patch-filters/issues/92 and single deleted line in https://github.com/checkstyle/patch-filters/issues/92#issuecomment-660469996 (single changed line has in it).

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/newline/defaultContextConfig.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+  <module name="FileTabCharacter">
+    <property name="eachLine" value="true"/>
+  </module>
+  <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
+    <property name="file" value="${tp}/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/newline/defaultContext.patch" />
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/newline/defaultContextConfig.xml:10]: Wrap long line, please apply to all configs

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/defaultContext.patch [hunk 2]
```diff
@@ -0,0 +1,22 @@
+diff --git a/Test.java b/Test.java
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/defaultContext.patch:1]: What for this file ?

All inputs (config, patch, input java files,... ) Should be separated into special folder and should be used by individual test run.

Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/defaultContext.patch]: removed

File: src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/newline/defaultContextConfig.xml (hunk 1 above)
Comment (HuGanghui) [reply src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchfilter/FileTabCharacter/MoveCodeToAnotherFileWithMinorChanges/newline/defaultContextConfig.xml]: done

**Comment by romani** [comment]
@HuGanghui , there is confclit please resolve.

**Pull request merged**